bin/qbatch

```diff
@@ -30,7 +30,10 @@ OPTIONS = [os.environ.get("QBATCH_OPTIONS")] if os.environ.get(
     "QBATCH_OPTIONS") else []
 
 if os.getenv('SGE_ROOT'):
-    SYSTEM = os.environ.get("QBATCH_SYSTEM", "sge")
+    if os.environ.get("QBATCH_SYSTEM", "sge") or os.environ.get("QBATCH_SYSTEM", "slurm"):
+        SYSTEM = os.environ.get("QBATCH_SYSTEM")
+    else:
+        sys.exit("specify either sge or slurm for QBATCH_SYSTEM")
```

Comment:
The auto detection systems are fragile anyways, we expect qbatch users to set their qbatch environment variables which  override detection.

Looking for sbatch is ok with me. The main challenge is reproducing the job chunking functionality so that qbatch acts the same everywhere.

_Sent from my OnePlus ONEPLUS A3000 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_